app/actions/StatusActions.ts

```diff
@@ -0,0 +1,50 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+export type SystemStatus = {
+  status: 'operational' | 'degraded' | 'major';
+  message: string;
+};
+
+type UptimeService = {
+  name: string;
+  status: 'up' | 'down' | 'degraded';
+  uptime: string;
+};
+
+export const fetchSystemStatus = createAsyncThunk(
+  'status/fetch',
+  async (): Promise<SystemStatus> => {
+    const response = await fetch(
```

Comment:
This is a very clean way of fetching, but it doesn't follow convention. Also, I'm pretty sure the `SystemStatus.FETCH` won't work now, meaning loading state doesn't get set.

I suppose it's fine if we don't need loading state, but in that case just remove the unused `SystemStatus.FETCH` action types.

Reply:
> This is a very clean way of creating a fetching thunk, but it doesn't follow lego-webapp convention.

It's sort of a mixture between "the convetion" and the readme fetching way. You're probably right about the loading states. Not sure what's the best way to do this is - do you have any input? I feel like you know our redux implementation _much_ better than I do